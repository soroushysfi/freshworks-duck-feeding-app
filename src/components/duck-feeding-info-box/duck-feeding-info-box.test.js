import { shallow, mount, render } from 'enzyme'
import DuckFeedingInfoBoxComponent from './duck-feeding-info-box.component'

let item;

beforeEach(() => {
     item = {feedingTime: '', duckCount: '', foodType: ''}
})
it('expect to match DuckFeedingInfoBoxComponent snapshot', () => {
    expect(shallow(<DuckFeedingInfoBoxComponent item={item}/>)).toMatchSnapshot()

})