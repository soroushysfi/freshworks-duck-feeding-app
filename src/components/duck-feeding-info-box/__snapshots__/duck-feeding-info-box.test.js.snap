// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to match DuckFeedingInfoBoxComponent snapshot 1`] = `
<div
  className="fetched-info-box"
>
  <span>
    Duck Feeding Time:
  </span>
  <p>
    NaN
    /
    NaN
    /
    NaN
  </p>
  <span>
    Duck Count:
  </span>
  <p />
  <span>
    Food Type:
  </span>
  <p />
</div>
`;
